scripts/contributor_dashboard_debug.py

```diff
@@ -0,0 +1,229 @@
+# coding: utf-8
```

Comment:
@sagangwee do you want this file to be owned by dev workflow or by the contributor experience team? It might make more sense for you all to own it so you can make sure any changes to it don't break your team's workflow

Reply:
I'm fine with owning this file.